### PR TITLE
Added support for the Rittal LCP DX Chiller

### PR DESCRIPTION
**Pull request by epiecs**
- Added support for the Rittal LCP DX Chiller (model 3311)
- Definition and discovery file has been created and tested on the Rittal Equipment
- Added a custom mib that can interpret the values
- Updated the documentation for Carel devices with the correct System and Enterprise OID needed for this change
- Made sure that the existing device is still supported

The MIB with additional docs can be found at: https://github.com/epiecs/carel-pco-mibs

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: mibs/carel/CAREL-RITTAL-LCP-3311-MIB.mib [hunk 1]
```diff
@@ -0,0 +1,767 @@
+-- *****************************************************************
```
Comment (Jellyfrog) [inline mibs/carel/CAREL-RITTAL-LCP-3311-MIB.mib:1]: Rename this file to `CAREL-RITTAL-LCP-3311-MIB` without .mib please

Comment (epiecs) [reply mibs/carel/CAREL-RITTAL-LCP-3311-MIB.mib]: done :)

**Comment by Jellyfrog** [comment]
> Please add test data so we can ensure your change is not broken in the future.
> 
> Read the docs to find out how: https://docs.librenms.org/Developing/os/Test-Units

@epiecs please check this

**Comment by epiecs** [comment]
> > Please add test data so we can ensure your change is not broken in the future.
> > Read the docs to find out how: https://docs.librenms.org/Developing/os/Test-Units
> 
> @epiecs please check this

Hi,

I will try to fix this today. I have captured the snmp data but our fw settings doesn't allow me to install snmpsim atm so I have to fix this first :)

Kr, Gregory

**Comment by epiecs** [comment]
Hi,

I received some errors when trying to run the save-test-data script. Capturing test data worked. I'll push a new commit with the test data included. Is it possible to help me with this issue?

```
librenms@SERVER:~$ ./scripts/save-test-data.php --os pcoweb-rittal-lcp-3311 --variant "Rittal LCP DX Chiller" --hostname 10.1.31.194
Starting snmpsim listening on 127.1.6.1:1161...
Traceback (most recent call last):
  File "/usr/bin/snmpsimd", line 576, in <module>
    '|'.join(log.gMap),
AttributeError: module 'snmpsim.log' has no attribute 'gMap'

Failed to start Snmpsim. Scroll up for error.
```

**Review by PipoCanaja — APPROVED** [review]
LGTM, thanx @epiecs

**Comment by Jellyfrog** [comment]
> Hi,
> 
> I received some errors when trying to run the save-test-data script. Capturing test data worked. I'll push a new commit with the test data included. Is it possible to help me with this issue?
> 
> ```
> librenms@SERVER:~$ ./scripts/save-test-data.php --os pcoweb-rittal-lcp-3311 --variant "Rittal LCP DX Chiller" --hostname 10.1.31.194
> Starting snmpsim listening on 127.1.6.1:1161...
> Traceback (most recent call last):
>   File "/usr/bin/snmpsimd", line 576, in <module>
>     '|'.join(log.gMap),
> AttributeError: module 'snmpsim.log' has no attribute 'gMap'
> 
> Failed to start Snmpsim. Scroll up for error.
> ```

Sounds like you installed it via your os package manager. Uninstall it and install it with `pip install snmpsim` instead

**Pull request merged**